docs/sg_execution_times.rst

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
Was this file automatically or is it intentional?

Reply:
The line change was intentional. When we migrate to pre-commit and ruff, we remove the automatic import sort that we have, Ruff doesn't have the isort with the configuration that we want (future, python default lib, simple library, complex library and our library). So I have to apply this to make things follow the pattern and clearer in the new documentation. I return isort on the pre-commit to avoid future issues with this.